app/models/pick.rb

```diff
@@ -17,4 +17,7 @@ def self.count_picks(event)
     end
     results
   end
+
+  def self.font_size(picks)
```

Comment:
Unused method argument - `picks`. If it's necessary, use `_` or `_picks` as an argument name to indicate that it won't be used. You can also write as `font_size(*)` if you want the method to accept any arguments but don't care about them.